lib/myxql.ex

```diff
@@ -147,10 +147,23 @@ defmodule MyXQL do
   MyXQL automatically disconnects the connection on the following error codes and they don't have
   to be configured:
 
-    * `ER_MAX_PREPARED_STMT_COUNT_REACHED`
+    * `:ER_MAX_PREPARED_STMT_COUNT_REACHED`
 
-  To convert error code number to error code name you can use `perror` command-line utility that
-  ships with MySQL client installation, e.g.:
+  You can pass error codes as integers too:
+
+      disconnect_on_error_codes: [1792]
+
+  ## Error codes
+
+  MyXQL maintains a mapping of atoms/integers for commonly used errors. You can add additional
+  ones by adding the following compile-time configuration:
+
+      config :myxql, :extra_error_codes, [
+        {1048, :ER_BAD_NULL_ERROR}
+      ]
+
```

Comment:
I think you need to tell them to do a mix deps.clean or something. See whatever we tell people to do on `plug/mime`. :)